### PR TITLE
SMG-45 HP->Squashhead, new sound

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
SMG 45 HP is replaced by new squashhead rounds.
New fire sound.
## Why It's Good For The Game
People kept yelling at me that the hollowpoints were useless so here you are, enjoy.
## Changelog
:cl:
add: New squashhead bullets for SMG-45
del: SMG-45 HP
soundadd: New firesound for SMG-45
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -630,6 +630,17 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	damage_falloff = 3
 	shrapnel_chance = 45
 
+/datum/ammo/bullet/smg/squash
+	name = "squash-head submachinegun bullet"
+	hud_state = "pistol_squash"
+	armor_type = "bomb"
```
Comment (Lumipharon) [inline code/modules/projectiles/ammo_datums.dm:636]: ?

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Review by Kuro020 — CHANGES_REQUESTED** [review]
we got discovered, just make it target bullet armor and do something else for it

**Review by Kuro020 — APPROVED** [review]

**Comment by srxtreme** [comment]
Woe Curse of weak ammo type upon ye

File: code/modules/projectiles/ammo_datums.dm [hunk 2]
```diff
@@ -630,6 +630,27 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
...
+	name = "squash-head submachinegun bullet"
+	hud_state = "pistol_squash"
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING
+	damage = 15
+	penetration = 15
+	armor_type = "bomb"
+	sundering = 1
+	damage_falloff = 2
+	shrapnel_chance = 0
+	var/shatter_duration = 3 SECONDS
```
Comment (ivanmixo) [inline code/modules/projectiles/ammo_datums.dm:643]: doc

**Review by ivanmixo — CHANGES_REQUESTED** [review]
also shatter on a full auto gun? buh

**Comment by Oscari-pog** [comment]
3 seconds of shatter is too long on a automatic gun

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**